cmake/onnxruntime_unittests.cmake

```diff
@@ -713,11 +713,17 @@ if (onnxruntime_BUILD_SERVER)
       set_source_files_properties("${TEST_SRC_DIR}/server/unit_tests/executor_test.cc" PROPERTIES COMPILE_FLAGS -Wno-unused-parameter)
     endif()
   endif()
-
+  if (onnxruntime_USE_OPENVINO_BINARY)
+    if ($ENV{INTEL_CVSDK_DIR} MATCHES "2019.3")
```

Comment:
this shouldn't be needed. it's already checked in CMakeLists.txt when openvino is enabled.